px-datetime-behavior.html

```diff
@@ -71,45 +46,6 @@
       var newMom = this.momentObj.clone().tz(this.timeZone);
       this.set('momentObj', newMom);
     }
-  },
-  /**
-   * This event is fired differently depending on config:
-   *  - If the px-datetime component has an Apply button, then it is fired when the
-   *    user clicks the Apply button (or press enter) AND the selected date/time is valid
-   *  - If the px-datetime component does not have an apply button, then it is fired
-   *    every time a change is made to the date/time AND the selected date/time is valid
-   *
-   * @event px-datetime-submitted
-   *
-   * @param {string} dateTime - The ISO 8601 string representing the date/time chosen by the user
-   * @param {object} momentObj - The moment.js object representing the date/time chosen by the user
-   */
-  _notifyDateTimeChanged: function(dateTime) {
-    if(dateTime !== undefined) {
-      //make sure we keep the moment synchronized in case moment has been changed externally
-      if(!this.momentObj || this.momentObj.toISOString() !== this.dateTime) {
-        this.set('momentObj', this._convertISOtoMoment(this.dateTime));
-      }
-
-      this.fire('px-datetime-submitted', {dateTime: this.dateTime, momentObj: this.momentObj});
```

Comment:
see comment on px-datetime-range-submitted

Reply:
It might make sense to fire a similar event to the one discussed about range in `px-datetime-field` after validation. They could also listen to `moment-obj-changed` and it would be the same thing.